conf/broker.conf

```diff
@@ -774,6 +774,9 @@ bookkeeperDiskWeightBasedPlacementEnabled=false
 # A value of '0' disables sending any explicit LACs. Default is 0.
 bookkeeperExplicitLacIntervalInMills=0
 
+  # Use older Bookkeeper wire protocol with bookie
+bookkeeperUseV2WireProtocol=true
```

Comment:
I think we already have this config in the broker.conf https://github.com/apache/pulsar/blob/master/conf/broker.conf#L680